test/integration/integration-test.js

```diff
@@ -323,30 +437,11 @@ describe('integration tests', function() {
 
 });
 
-function createAccount(api, address) {
-  const root = 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh';
-  const secret = 'snoPBrXtMeMyMHUVTgbuqAfg1SUTb';
-  const amount = {
-    currency: 'XRP',
-    value: '10000'
-  };
-  return api.preparePayment(root, {
-    source: {address: root, maxAmount: amount},
-    destination: {address, amount}
-  }).then(prepared => {
-    return api.submit(api.sign(prepared.txJSON, secret).signedTransaction);
-  }).then(() => {
-    return acceptLedger(api);
-  });
-}
-
-describe.skip('integration tests - standalone rippled', function() {
+describe('integration tests - standalone rippled', function() {
   const instructions = {maxLedgerVersionOffset: 10, fee: '1'};
```

Comment:
Oops, I forgot to delete the "fee" here, can you delete it? I think it should work now without it.